src/metabase/app_db/custom_migrations.clj

```diff
@@ -1823,3 +1823,12 @@
 
 (define-migration MoveExistingAtSymbolUserAttributes
   (reserve-at-symbol-user-attributes/migrate!))
+
+(define-migration BackfillTransformTargetDbId
+  (doseq [{:keys [id target]} (t2/query {:select [:id :target]
```

Comment:
tho I know users might have lots of transform at this point but it's good practice to make these performance and not realizing the whole transform table here. turns this into a `t2/reduicible-query` and we gtg here I think.